src/plugins/recordTypes/person/fields.js

```diff
@@ -1259,31 +1270,35 @@ export default (configContext) => {
                 },
               },
             },
-            suppliedBirthDate: {
+            suppliedStructuredBirthDateGroup: {
               [config]: {
+                dataType: DATA_TYPE_STRUCTURED_DATE,
                 messages: defineMessages({
                   fullName: {
-                    id: 'field.persons_common.suppliedBirthDate.fullName',
-                    defaultMessage: 'Birth date supplied',
+                    id: 'field.persons_common.suppliedStructuredBirthDateGroup.fullName',
+                    defaultMessage: 'Supplied structured birth date',
+                  },
+                  groupName: {
+                    id: 'field.persons_common.suppliedStructuredBirthDateGroup.groupName',
+                    defaultMessage: 'Date',
```

Comment:
groupName should in most cases be the same as name, so probably want "Supplied" here to match the name.